README.md

```diff
@@ -251,6 +253,178 @@ To run other tests:
 ./scripts/run_tests.sh
 ```
 
+## Deployment
+
+Instructions on how to set up the pipeline on a production environment.
```

Comment:
```suggestion
Instructions on how to set up the pipeline on a production environment manually. The pipeline can also be installed with Puppet using the [pmc_pipeline](https://github.com/thehyve/pmc_pipeline) module.
```

Reply:
This is still a private repository. Maybe we can discuss making it a public one.

Reply:
Good point. Good to leave it like it was.